advocacy_docs/pg_extensions/wait_states/installing.mdx

```diff
@@ -3,12 +3,41 @@ title: Installing EDB Wait States
 navTitle: Installing
 ---
 
-## On Linux 
+## On all Linux systems except Debian and Ubuntu
 
-To install EDB Wait States on a Linux system:
+To install EDB Wait States on any Linux operating system except Debian or Ubuntu:
 
-1.  Install EDB Wait States with the `edb-asxx-server-edb_wait_states` RPM package, where `xx` is the EDB Postgres Advanced Server version number.
+1.  Enter the following command:
 
+  ```shell
+  sudo <package-manager> -y install <install-command>
+  ```
+
+  Where: 
+  
+    - `<package-manager>`is the package manager used with your operating system:
+
+      | Package manager | Operating system     |
+      | --------------- | -------------------------------- |
+      | dnf             | RHEL 8/9 and derivatives         |
+      | yum             | RHEL 7 and derivatives, CentOS 7 |
+      | zypper          | SLES                             |
+
+    - `<install-command>` is the command required for the type of Postgres distribution you're using. In these commands `<version>` is the version of the Postgres distribution. 
+
+      | Type of Postgres distribution | Install command                             |
+      | ----------------------------- | ------------------------------------------- |
+      | PostgreSQL                    | postgresql`<version>`-edb-wait-states       |
```

Comment:
This naming is still incorrect. On RHEL systems (rpm based) there are underscores instead of dashes
```
$ cs.sh enterprisedb/enterprise 'name:postgresql16-edb_wait_states version:1.1.0*'
Getting list of packages ... OK
enterprisedb/enterprise name: postgresql16-edb_wait_states-debuginfo-1.1.0-1.el7.x86_64.rpm upload time: 2023-11-09T10:49:37.183668Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.el8.x86_64.rpm upload time: 2023-11-09T10:49:37.176887Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-debuginfo-1.1.0-1.el9.ppc64le.rpm upload time: 2023-11-09T10:49:37.135491Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.el7.x86_64.rpm upload time: 2023-11-09T10:49:36.994248Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.sles15sp4.x86_64.rpm upload time: 2023-11-09T10:49:36.948784Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.el9.ppc64le.rpm upload time: 2023-11-09T10:49:36.942372Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-debuginfo-1.1.0-1.el9.x86_64.rpm upload time: 2023-11-09T10:49:36.843485Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.sles12sp5.x86_64.rpm upload time: 2023-11-09T10:49:36.708851Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-debuginfo-1.1.0-1.el7.ppc64le.rpm upload time: 2023-11-09T10:49:36.624840Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-debuginfo-1.1.0-1.el8.x86_64.rpm upload time: 2023-11-09T10:49:36.559505Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.el8.ppc64le.rpm upload time: 2023-11-09T10:49:36.540351Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.sles15sp4.ppc64le.rpm upload time: 2023-11-09T10:49:36.527113Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-debuginfo-1.1.0-1.el8.ppc64le.rpm upload time: 2023-11-09T10:49:36.426708Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.sles12sp5.ppc64le.rpm upload time: 2023-11-09T10:49:36.361250Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.el7.ppc64le.rpm upload time: 2023-11-09T10:49:36.341814Z
enterprisedb/enterprise name: postgresql16-edb_wait_states-1.1.0-1.el9.x86_64.rpm upload time: 2023-11-09T10:49:36.267677Z
```

This is how I installed the extension on a Rocky 8 server:

```
dnf install postgresql16-edb_wait_states.x86_64
```

Reply:
right,
on RPM - edb-as16-server-edb_wait_states
on Debian/Ubuntu- edb-as16-server-edb-wait-states

Reply:
@martinmarques @tusharahuja Can you take a look and confirm that what we want is underscores for PostgreSQL? Thanks.

Reply:
Tushar has approved the changes so I am going to mark this as resolved. @martinmarques let me know if you have further thoughts